src/EnergyPlus/General.cc

```diff
@@ -972,7 +972,9 @@ void Iterate(Real64 &ResultX,  // ResultX is the final Iteration result passed b
 
         // New guess calculated from LINEAR FIT of most recent two points
         DY = Y0 - Y1;
-        if (std::abs(DY) < small) DY = small;
+        if (std::abs(DY) < small) {
+            DY = small;
+        }
```

Comment:
Unrelated, but it's handy to put a debugger breakpoint... (We should have clang-format do that everywhere IMHO)

Reply:
Agreed. It's standard practice but not enforced.